Automattic/woocommerce-payments
# fix: missing translations, improved a11y on new testing instructions

Fixes https://github.com/Automattic/woocommerce-payments/issues/9399

#### Changes proposed in this Pull Request









The new testing instructions were missing some translations wrappers.
The button missed an `:active` style, which provides feedback when a button is clicked.
The button didn't have a big enough click target, so I added some `padding` to increase it.

I moved the icon into a separate `<i>` element, because it allows us to scale the icon on `:active` without scaling the button itself.
![2024-08-19 11 42 14](https://github.com/user-attachments/assets/0c70267a-2e98-481b-85c1-d9c7493e6045)
![2024-08-19 11 38 40](https://github.com/user-attachments/assets/38dedf44-9c79-4998-a07e-e2dba1e4936c)

I also made the solution work with payment methods other than card, like SEPA.
<img width="543" alt="Screenshot 2024-08-19 at 11 42 45 AM" src="https://github.com/user-attachments/assets/d84d0d83-5956-4f04-a9f0-40f2ab4d086b">

I also added a toast notification for fun on blocks checkout.
<img width="382" alt="Screenshot 2024-08-19 at 12 09 11 PM" src="https://github.com/user-attachments/assets/70279421-3ba3-4217-8206-a22700e5048e">

#### Testing instructions





- Ensure WooPayments is in test mode
- Go to block or shortcode checkout
- Clicking on the icon next to the card number should copy the card number to your clipboard

-------------------


**Post merge**
